### DISCUSSION TITLE
How can one use the "xcolor" latex package in manim Tex?

### DISCUSSION THREAD

**Discussion by Quantum1352** [3b1b/manim#1996] [Q&A]
Hi,

I want to have the labels of the tasks environment in blue. The following is the code I am using. Manim does not produce any errors, but the labels are still in white. Help will be appreciated.

Regards 

from manim import *

class OVQ(Scene):
        def construct(self):
            myTemplate = TexTemplate()
            myTemplate.add_to_preamble(r"\usepackage{tasks}")
            myTemplate.add_to_preamble(r"\usepackage{xcolor}")
            
            t1 = Tex(r"""A Sample Text A Sample Text A Sample Text""",r"""
            \begin{tasks}[label=\color{blue}\Alph*)]
            \task The First Task
            \task The Second Task
            \end{tasks}""", 
                      tex_template=myTemplate,
                      tex_environment=r"\begin{tabular}{p{9.6 cm}}"
                      )

           
            self.play(Write(t1))
            
<img width="1605" alt="Screenshot 2023-02-18 at 23 32 50" src="https://user-images.githubusercontent.com/106308764/219902333-4de8cca8-3095-40bd-92bd-0c38af17c5e4.png">

**Comment by TonyCrane** [comment]
Maybe manim always set a default color to Tex.

But you are using the community version of manim, so I'm not really clear about it. You'd better ask this question on their discord channel or open an issue at ManimCommunity/manim instead of here.

**Reply by Quantum1352** [reply]
Thanks for the help!
